doc/ansible/dev/apb_migration_guide.md

```diff
@@ -0,0 +1,241 @@
+
+In your `build/Dockerfile`, ensure that your playbooks and roles are being copied to `${HOME}/roles` and `${HOME}/playbooks`, and that your `watches.yaml` is being copied to `${HOME}/watches.yaml`. 
+
+If you are installing any additional dependencies, ensure that those are reflected in your `build/Dockerfile` as well. 
+
+As a sample, your `build/Dockerfile` will probably look something like this:
+
+```
+FROM quay.io/operator-framework/ansible-operator:v0.9.0
+
+COPY watches.yaml ${HOME}/watches.yaml
+
+COPY roles/ ${HOME}/roles/
+COPY playbooks/ ${HOME}/playbooks/
+```
+
+Once this is done you may remove your original APB `Dockerfile`.
+
+#### watches.yaml
+In the `watches.yaml`, ensure the playbook for your `kind` points to your `provision.yml` playbook in the container (likely location for that will be `/opt/ansible/playbooks/provision.yml`). 
+
+Next, add a finalizer block with a name of: `finalizer.<name>.<group>/<version>`, and set the playbook to point to your `deprovision.yml` in the container (likely location for that will be `/opt/ansible/playbooks/deprovision.yml`). For the memcached-operator we generated above, the watches.yaml would look like this:
+
+```yaml
+---
+- version: v1alpha1
+  group: apps.example.com
+  kind: Memcached
+  playbook: /opt/ansible/playbooks/provision.yml
+  finalizer:
+    name: finalizer.memcached.apps.example.com/v1alpha1
+    playbook: /opt/ansible/playbooks/deprovision.yml
+```
+
+##### Binding
+If you have a `bind` playbook, add a new entry to your `watches.yaml` (you can copy paste the existing entry). 
+
+The `version` and `group`, will remain unchanged, but update the `kind` with a `Binding` suffix. 
+
+For example, if you have a resource with `kind: Memcached`, the kind of your new entry will be `MemcachedBinding`. 
+
+The playbook for this entry should map to your `bind` playbook, (likely location `/opt/ansible/playbooks/bind.yml`), and if you have an `unbind` playbook then set the playbook for the finalizer to point to it (likely location `/opt/ansible/playbooks/unbind.yml`). If you don't have an `unbind` playbook, remove the finalizer block for your `Binding` resource.
+
+For an APB with both `bind` and `unbind` playbooks, the `watches.yaml` would end up looking like this:
+
+```yaml
+---
+- version: v1alpha1
+  group: apps.example.com
+  kind: Memcached
+  playbook: /opt/ansible/playbooks/provision.yml
+  finalizer:
+    name: finalizer.memcached.apps.example.com/v1alpha1
+    playbook: /opt/ansible/playbooks/deprovision.yml
+- version: v1alpha1
+  group: apps.example.com
+  kind: MemcachedBinding
+  playbook: /opt/ansible/playbooks/bind.yml
+  finalizer:
+    name: finalizer.memcachedbinding.apps.example.com/v1alpha1
+    playbook: /opt/ansible/playbooks/unbind.yml
+```
+
+
+You will also need to run `operator-sdk add crd --api-version=<group>/<version> --kind=<kind>` to generate a new CRD and example in `deploy/crds`.
+
+#### deploy/crds/
```

Comment:
Move this heading above the line that reads:
> You will also need to run ....

Otherwise, I feel that step is easily missed and someone will forget to generate their CRDs.

Reply:
What should the user expect to see in that `deploy/crds` directory? yaml files named crds? something else? Just point out what should be there so they can tell whether the command ran successfully.